driver/collector/postgres_collector.py

```diff
@@ -320,6 +322,29 @@
     i.indrelid, i.indexrelid;
 """
 
+# query digest id not included pre pg 14
+LONG_RUNNING_QUERY_NO_ID_SQL_TEMPLATE = """
+SELECT
+    pid, query_start, datid, datname, usename, state
+FROM
+    pg_stat_activity
+WHERE
+    query_start < {lr_query_start_timestamp};
+LIMIT
+    {n};
+"""
+
+LONG_RUNNING_QUERY_SQL_TEMPLATE = """
+SELECT
+    pid, query_id, query_start, datid, datname, usename, state
```

Comment:
I don't think we need to collect username here.

We also need these fields: wait_event, backend_type, xact_start, state_change

Reply:
wait_event_type

Reply:
Added the proposed fields